test/e2e/docker/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
 # instead of spending time compiling them.
-FROM golang:1.15
+FROM golang:1.17
 
 RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
-RUN apt-get -qq install -y cmake sudo libgmp-dev libleveldb-dev librocksdb-dev >/dev/null
+RUN apt-get -qq install -y cmake sudo libgmp-dev libleveldb-dev librocksdb-dev python3-dev >/dev/null
```

Comment:
Do we also need to update DOCKER/Dockerfile* ? These are used to build production containers